internal/webhook/csv.go

```diff
@@ -0,0 +1,110 @@
+package webhook
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/go-logr/logr"
+	opv1a1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"k8s.io/utils/ptr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	WebhookPath = "/mutate-operators-coreos-com-v1alpha1-clusterserviceversion"
```

Comment:
this doesn't need to be GVK identifiable by any means, not user facing, longer names are prone to typos and short name would suffice, how about '/mutate-csv' or can expand csv.